lib/tasks/webpacker.rake

```diff
@@ -97,6 +97,43 @@ namespace :webpacker do
 
       exec './bin/yarn add --dev typescript ts-loader && ./bin/yarn add "core-js zone.js rxjs @angular/core @angular/common @angular/compiler @angular/platform-browser @angular/platform-browser-dynamic"'
     end
+
+    desc "Install everything needed for Vue"
+    task :vue do
+      config_path = Rails.root.join('config/webpack/shared.js')
+      config = File.read(config_path)
+
+      # Module resolution https://webpack.js.org/concepts/module-resolution/
+      if config.include?("'vue$':'vue/dist/vue.common.js'")
```

Comment:
@zigomir thanks for informing me. I guess it needs to be updated to  `'vue$': 'vue/dist/vue.esm.js'` 

Will send in a bundle PR after I read through the entire vue 2.2.x release.